docs/sources/datasources/pyroscope/_index.md

```diff
@@ -37,13 +41,13 @@ In this case, the screenshot shows memory profiles alongside panels for logs and
 
 ## Visualize traces and profiles data using Traces to profiles
 
-You can link profile and tracing data using your Pyroscope data source with the Tempo data source.
+You can link profile and tracing data using your Pyroscope data source with the Tempo data source. To learn more about how profiles and tracing can work together, refer to [Profiling and tracing synergies](./profiling-tracing-intro).
```

Comment:
[Semantic line breaks](https://sembr.org/) are better for future line based diffing in the GitHub UI and result in the same HTML output.

```suggestion
You can link profile and tracing data using your Pyroscope data source with the Tempo data source.
To learn more about how profiles and tracing can work together, refer to [Profiling and tracing synergies](./profiling-tracing-intro/).
```